src/MBrace.Flow/CloudFlow.fs

```diff
@@ -1123,3 +1139,55 @@ module CloudFlow =
                         return! projection collector.Result
                 } }
 
+
+
+
+                
+    /// <summary>Creates a CloudFlow from the ReceivePort of a CloudChannel</summary>
+    /// <param name="channel">the ReceivePort of a CloudChannel.</param>
+    /// <returns>The result CloudFlow.</returns>
+    let ofCloudChannel (channel : IReceivePort<'T>) : CloudFlow<'T> =
```

Comment:
One way to specify the degree of parallelism
https://github.com/palladin/MBrace.Core/blob/master/tests/MBrace.Flow.Tests/CloudFlowTests.fs#L476
another idea is to have an overload to CloudFlow,ofCloudSource